src/modules/verifiable-credentials/types/verifiable-credentials.types.ts

```diff
@@ -18,6 +18,7 @@ export interface RoleCredentialSubjectParams {
 export interface ProofOptions {
   verificationMethod?: string;
   proofPurpose?: string;
+  challenge: string;
```

Comment:
```suggestion
  challenge?: string;
```
I don't think we need `challenge` for all cases